### DISCUSSION TITLE
Pluggin socketio server into an existing eventloop

### DISCUSSION THREAD

**Discussion by wishWinds** [miguelgrinberg/python-socketio#1217] [Q&A]
My socketio server is act as a plugin with other module like names **A**. 
**A** has create an event loop for long run, How can i create my socketio server embedded in existing event loop?

**Comment by miguelgrinberg** [comment]
This package does not provide a web server component. You will need to choose a web server from the list of supported ones that can work in the way that you need.

**Comment by wishWinds** [comment]
Hi. Thanks.

I use the aiohttp from your documentation. 

But how to embedded the `app` into my existing event loop?

**Comment by miguelgrinberg** [comment]
Have you consulted the aiohttp documentation regarding this? I don't use this framework much, but https://docs.aiohttp.org/en/stable/web_advanced.html#application-runners seems to be related to what you need.
